src/git-lfs/install.sh

```diff
@@ -185,6 +186,36 @@ if [ "${use_github}" = "true" ]; then
     install_using_github
 fi
 
+# --- Generate a 'pull-git-lfs-artifacts.sh' script to be executed by the 'postCreateCommand' lifecycle hook
+
+tee /tmp/pull-git-lfs-artifacts.sh > /dev/null \
+<< EOF
+#!/bin/sh
+set -e
+AUTO_PULL=${AUTO_PULL}
+EOF
+
+tee -a /tmp/pull-git-lfs-artifacts.sh > /dev/null \
+<< 'EOF'
+
+echo "Fetching git lfs artifacts..."
+
+if [ "${AUTO_PULL}" != "true" ]; then
+    echo "(!) Skipping 'git lfs pull' because 'autoPull' is not set to 'true'"
+    exit 0
+fi
+
+# Check if repo is a git lfs repo.
+if ! git lfs ls-files > /dev/null 2>&1; then
+    echo "(!) Skipping automatic 'git lfs pull' because no git lfs files were detected"
+    exit 0
+fi
+
+git lfs pull
+EOF
+
+chmod 755 /tmp/pull-git-lfs-artifacts.sh
```

Comment:
Question, should this be `chowned` for the user (assuming it's not `root`)

Reply:
good point, i'll add that and a test with a non-root remote user

Reply:
well actually, these permissions let anyone read or execute so chowning is unnecessary.  I'll still add a remoteUser to validate that.